mongodb/files/mongodb.conf

```diff
@@ -30,7 +30,7 @@ replSet = {{ server.replica_set }}
 auth = true
 
 # Verbose logging output.
-verbose = true
+# verbose = true
```

Comment:
This option is not to set log verbosity, this is switch to enable maximum logging. To set logging level there is an `oplog` attribute that is not managed and there is no reasoning to work on that as newer version uses different configuration structure at all.

This PR is hotfix.

Verbose (true/false) should be by default switched off. There is no reason to make this configurable as it's used only for manual debugging.

Newer version of mongodb  uses different configuration files, new style is not even compatible with the current options - so attribute model would not fit anyway: https://docs.mongodb.com/manual/reference/configuration-options/#systemlog-options